### ISSUE TITLE
Why "LIDA depends on openAI"?

### ISSUE THREAD

**Issue by alexaios** [microsoft/lida#116]
As mentioned in the readme file, "LIDA depends on llmx and openai". Just wanted to understand which part of lida has to use openAI?
According to the following description, if I use other llm models as text generator which will generate summary, goals, code to visualize, and in this approach does LIDA still depends on openAI? Can someone please elaborate? 

``` 
LIDA uses the [llmx](https://github.com/victordibia/llmx) library as its interface for text generation. llmx supports multiple local models including HuggingFace models. You can use the huggingface models directly (assuming you have a gpu) or connect to an openai compatible local model endpoint e.g. using the excellent [vllm](https://vllm.readthedocs.io/en/latest/) library.
```

**Comment by skyprince999** [comment]
No. LIDA doesnt depend on OpenAI alone. You can use other LLMs as long as they use the OpenAI syntax. 

Sample code for using other LLMs is provided in the tutorial notebook. 

https://github.com/microsoft/lida/blob/main/notebooks/tutorial.ipynb
